go.mod

```diff
@@ -178,8 +178,8 @@ require (
 )
 
 replace (
-	// CVE fixes for https://avd.aquasec.com/nvd/2024/cve-2024-45337
-	golang.org/x/crypto => golang.org/x/crypto v0.31.0
+	// CVE fixes for https://avd.aquasec.com/nvd/cve-2025-22869
+	golang.org/x/crypto => golang.org/x/crypto v0.35.0
```

Comment:
Why does this need to be a replace and not just in requires?

Reply:
it's an indirect dependency that's being pulled in.

Reply:
Can't we still update the dependency in require though? Using replace means we need to remember to bump it in future manually...

Reply:
Sure.